source/CardGames.py

```diff
@@ -274,6 +282,7 @@ def Play(dealer: Dealer, players: list, pot: Pot):
         dealer.dealCards(1, [player])
         print("Card drawn by %s.\n" % (player.name))
         player.display()
+        checkIfEasterEgg(player)
```

Comment:
Perfect insertion into the Play class. My only thought is will the easter egg interfere with the playthrough.

Reply:
i didn't change anything in the functionality that already exists. this will simply run in addition to the main loop of the game.

Reply:
Perfect! Great work!